onnx/reference/ops/op_split_to_sequence.py

```diff
@@ -37,7 +37,7 @@ def common_run(
 
     def _run(self, mat, split=None, axis=None, keepdims=None):  # type: ignore
         res = self.common_run(mat, split, axis=axis)
-        if keepdims == 0 and axis is not None:
+        if keepdims == 0:
             for i in range(len(res)):
```

Comment:
Consider enumerate?

Reply:
I chose this syntax because I modify the list while walking through it.